doc/man/bitcoin-cli.1

```diff
@@ -22,6 +22,10 @@ This help message
 .IP
 Specify configuration file (default: bitcoin.conf)
 .HP
+\fB\-netconf=\fR<file>
```

Comment:
I wonder - is there any practical point in being able to override this?

Reply:
Aren't the doc/man/ files automatically generated anyway so shouldn't be modified directly?

Reply:
@MeshCollider Yep, that's correct; worse the help text for bitcoin-cli is missing. Will fix.